site-templates/pages/appRegistration.gomd

```diff
@@ -0,0 +1,22 @@
+<div class="row">
+
+  <div class="col-md-8 content">
+  
+    <h1>Application Registration</h1>
+
+    <h3>Overview</h3>
+    <p>
+    Application Registration enables a user to register for a Bluescape's account and to create an application in Bluescape. Upon creating an application,
```

Comment:
I think we remove the mention of "enables a user to register for a Bluescape account"

Reply:
What would be a good introduction sentence if we remove that? "To create a Bluescape's application...?

Reply:
Yeah, I think telling them "App Registration is used for registering for a Bluescape account" might confuse them